iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -97,13 +98,15 @@ public static Table loadTable(Configuration conf, Properties props) {
             props.getProperty(InputFormatConfig.CATALOG_NAME));
   }
 
-  private static Table loadTable(Configuration conf, String tableIdentifier, String tableLocation,
-                                 String catalogName) {
+  private static Table loadTable(Configuration conf, String tableIdentifier, String tableLocation, String catalogName) {
     Optional<Catalog> catalog = loadCatalog(conf, catalogName);
 
+    Table cachedTable = null;
```

Comment:
Are these changes still necessary?

Reply:
Not at all. The revert was not 100% successful.